cleanlab/filter.py

```diff
@@ -235,7 +236,19 @@ class 0, 1, ..., K-1.
 
     # Number of examples in each class of labels
     if multi_label:
-        label_counts = value_counts([i for lst in labels for i in lst])
+        return _find_label_issues_multilabel(
```

Comment:
To remain backwards compatible, we also have to additionally support the current case where `multi_label=True` and `confident_joint` is k x k matrix. My suggestion is to explicitly check for this shape of `confident_joint` inside `_find_label_issues_multilabel()` and if it is detected then:

-  set `confident_joint = None` internally at the start of the function (so it is completely ignored subsequently).
- raise Warning that says: "The new recommended format for `confident_joint` in multi_label settings is .... (as output by `compute_confident_joint(...,multi_label=True)`).  Your k x k `confident_joint` in the old format is being ignored.